### ISSUE TITLE
执行：ansible-playbook start.yml 报错，可能是什么问题导致的？

### ISSUE THREAD

**Issue by Berny-Chen** [pingcap/tidb-ansible#232]
TASK [wait up] ****************************************************************************
fatal: [tidb1]: FAILED! => {"changed": false, "elapsed": 300, "msg": "Timeout when waiting for search string 200 OK in tidb1:2379"}
Ask for help:
Contact us: sre@pingcap.com
It seems that you encounter some problems. You can send an email to the above email address, attached with the tidb-ansible/inventory.ini and tidb-ansible/log/ansible.log files and the error message, or new issue on https://github.com/pingcap/tidb-ansible/issues. We'll try our best to help you deploy a TiDB cluster. Thanks. :-)
        to retry, use: --limit @/opt/src/tidb-ansible/retry_files/start.retry
PLAY RECAP ********************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0   
tidb1                      : ok=18   changed=0    unreachable=0    failed=1   
tidb2                      : ok=15   changed=0    unreachable=0    failed=0   
tidb3                      : ok=15   changed=0    unreachable=0    failed=0   
tidb4                      : ok=15   changed=0    unreachable=0    failed=0   

ERROR MESSAGE SUMMARY *********************************************************************
[tidb1]: Ansible FAILED! => playbook: start.yml; TASK: wait up; message: {"changed": false, "elapsed": 300, "msg": "Timeout when waiting for search string 200 OK in tidb1:2379"}

**Comment by LinuxGit** [comment]
Clould you provide your inventory.ini and the output of `sudo systemctl status pd.service` on your PD server ?

**Comment by Berny-Chen** [comment]
# TiDB Cluster Part
[tidb_servers]
tidb1

[tikv_servers]
tidb2
tidb3
tidb4

[pd_servers]
tidb1

[spark_master]

[spark_slaves]

# Monitoring Part
[monitoring_servers]
tidb1

[grafana_servers]
tidb1

[monitored_servers:children]
tidb_servers
tikv_servers
pd_servers
spark_master
spark_slaves

## Binlog Part
[pump_servers:children]
tidb_servers

[drainer_servers]

[pd_servers:vars]
# location_labels = ["zone","rack","host"]

## Global variables
[all:vars]
deploy_dir = /data/tidb/deploy

## Connection
# ssh via root:
# ansible_user = root
# ansible_become = true
# ansible_become_user = tidb
# ssh via normal user
ansible_user = tidb

cluster_name = test-cluster

tidb_version = v1.0.7

# deployment methods, [binary, docker]
deployment_method = binary

# process supervision, [systemd, supervise]
process_supervision = systemd

# timezone of deployment region
timezone = Asia/Shanghai
set_timezone = True

# misc
enable_firewalld = False
# check NTP service
enable_ntpd = True
machine_benchmark = False
set_hostname = False

# binlog trigger
enable_binlog = False
# zookeeper address of kafka cluster, example:
# zookeeper_addrs = "192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181"
zookeeper_addrs = ""

**Comment by Berny-Chen** [comment]
inventory.ini:
```
# TiDB Cluster Part
[tidb_servers]
tidb1

[tikv_servers]
tidb2
tidb3
tidb4

[pd_servers]
tidb1

[spark_master]

[spark_slaves]

# Monitoring Part
[monitoring_servers]
172.19.6.247

[grafana_servers]
172.19.6.247

[monitored_servers:children]
tidb_servers
tikv_servers
pd_servers
spark_master
spark_slaves

## Binlog Part
[pump_servers:children]
tidb_servers

[drainer_servers]

[pd_servers:vars]
# location_labels = ["zone","rack","host"]

## Global variables
[all:vars]
deploy_dir = /data/tidb/deploy
## Connection
# ssh via root:
# ansible_user = root
# ansible_become = true
# ansible_become_user = tidb

# ssh via normal user
ansible_user = tidb

cluster_name = test-cluster

tidb_version = v1.0.7

# deployment methods, [binary, docker]
deployment_method = binary

# process supervision, [systemd, supervise]
process_supervision = systemd

# timezone of deployment region
timezone = Asia/Shanghai
set_timezone = True

# misc
enable_firewalld = False
# check NTP service
enable_ntpd = True
machine_benchmark = False
set_hostname = False

# binlog trigger
enable_binlog = False
# zookeeper address of kafka cluster, example:
# zookeeper_addrs = "192.168.0.11:2181,192.168.0.12:2181,192.168.0.13:2181"
zookeeper_addrs = ""

# KV mode
deploy_without_tidb = False
```
OS version:CentOS 7.4

**Comment by LinuxGit** [comment]
@Berny-Chen Please add your ip address under tidb_servers、tikv_servers or pd_servers in `inventory.ini` file. See https://github.com/pingcap/docs-cn/blob/master/op-guide/ansible-deployment.md.

**Comment by Berny-Chen** [comment]
thinks,let me try again

**Comment by LinuxGit** [comment]
OK.  You can new or reopen issue if you have other questions.
